playbooks/roles/common_vars/defaults/main.yml

```diff
@@ -14,13 +14,17 @@ COMMON_BASIC_AUTH_EXCEPTIONS:
 # Settings to use for calls to edxapp manage.py
 COMMON_EDXAPP_SETTINGS: 'aws'
 
+# Set to True to install aws role when running edx_sandbox playbook.
+COMMON_ENABLE_AWS_INTEGRATION: False
```

Comment:
@Metfriet, there has to be a corresponding PR against the `configuration-secure` repository to set this to `True`.